pkgs/applications/emulators/mame/default.nix

```diff
@@ -138,18 +140,26 @@ stdenv.mkDerivation rec {
     runHook preInstall
 
     make -f dist.mak PTR64=${lib.optionalString stdenv.is64bit "1"}
+
+    # mame
     mkdir -p ${dest}
     mv build/release/*/Release/mame/* ${dest}
 
-    mkdir -p $out/bin
-    find ${dest} -maxdepth 1 -executable -type f -exec mv -t $out/bin {} \;
-    install -Dm755 src/osd/sdl/taputil.sh $out/bin/taputil.sh
```

Comment:
Removed since other distros (e.g. Arch) doesn't package it and it is a really simple script that only works in a very specific case. If someone wants to use it, I think they can grab from the source code.